bindings/python/tests/tenant_tests.py

```diff
@@ -59,6 +59,10 @@ def test_tenant_operations(db):
     fdb.tenant_management.create_tenant(db, b'tenant1')
     fdb.tenant_management.create_tenant(db, b'tenant2')
 
+    tenant_list = fdb.tenant_management.list_tenants(db, b'a', b'z', 10).to_list()
+    assert tenant_list[0] == b'tenant1'
+    assert tenant_list[1] == b'tenant2'
```

Comment:
This could be extended to check key prefixes similar to the examples below that are parsing the JSON document. That way you could verify that the value matches expectation as well.